### PR TITLE
add a module to associate aws_iam_access_key credentials to a tfe wor…

### PR DESCRIPTION
**Pull request by JoseD92**
I add a module for associating a aws_iam_access_key credentials to a tfe workspace

### CODE REVIEW

File: modules/tf-cloud-credential/variables.tf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+variable "workspace_name_prefix" {
```
Comment (ketzacoatl) [inline modules/tf-cloud-credential/variables.tf]: This would probably be better as `name_prefix`.

File: modules/tf-cloud-credential/variables.tf [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  type        = string
+  description = "The workspace organization"
+}
+
+variable "iam_access_key" {
+  type = object({
+    id     = string
+    secret = string
+  })
+  description = "The aws_iam_access_key to use as credentials for workspace."
```
Comment (ketzacoatl) [inline modules/tf-cloud-credential/variables.tf]: Should be: `"The aws_iam_access_key id/secret pair to use as credentials for the workspace."`

File: modules/tf-cloud-credential/variables.tf (hunk 1 above)
Comment (mcgirr) [reply modules/tf-cloud-credential/variables.tf]: I resolved this in the latest commit for this PR

File: modules/tf-cloud-credential/variables.tf (hunk 2 above)
Comment (mcgirr) [reply modules/tf-cloud-credential/variables.tf]: I also resolved this in the latest commit for this PR

**Comment by mcgirr** [comment]
@JoseD92 I didn't want to hold up this PR but in the future when you make a PR for `terraform-aws-foundation` please be sure to update the release changelog with the changes the PR makes. Especially in this case since it adds a new module.

**Comment by JoseD92** [comment]
Thanks @mcgirr, I will do next time.

**Pull request merged**